src/gc/gcenv.ee.standalone.inl

```diff
@@ -11,166 +11,196 @@
 // will be fowarded to this interface instance.
 extern IGCToCLR* g_theGCToCLR;
 
+// A note about this:
+// In general, we don't want to pretend to be smarter than the compiler
+// and force it to inline things. However, inlining is here is required
+// for correctness as it stands today (though it will not always be required).
```

Comment:
Right. It may be useful to actually start building the standalone GC as .dll / .so instead of the approximation used today (where you build it as standalone, but still link statically). The benefits are:

- You do not need hacks like these.
- You can add it to the build by default. No need to have a separate build flavor to make sure that it builds. The shipping CoreCLR .dll/.so would still be building with statically linked flavor of the GC.
- I expect that you would need to add a bunch of stubbed out methods to get the standalone .dll / .so to link. It would give you a clear picture and burn down rate about what all you still need to deal with to get to a fully working one.